launch/test/counter.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import argparse
+import signal
+import sys
+import time
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description=(
+            'Simple program outputting a counter. '
+            'Even values go to STDERR, odd values go to STDOUT.'))
```

Comment:
Without them the indentation does not follow PEP8. Or the second line would need to start aligned to the equal sign.